opentelemetry/proto/resource/v1/resource.proto

```diff
@@ -42,4 +42,21 @@ message Resource {
   //
   // Status: [Development]
   repeated opentelemetry.proto.common.v1.EntityRef entity_refs = 3;
+
+  // Set of attributes that describe the resource.
+  // Key and Value combination MUST be unique.
+  //
+  // Note: This is currently used exclusively in the Profiling signal.
+  //
+  // Status: [Development]
+  repeated StringKeyValue = 4;
```

Comment:
This is missing a field name. You can merge https://github.com/florianl/opentelemetry-proto/pull/3 to fix it if you like.